contracts/erc20-bridge-sampler/contracts/src/DummyLiquidityProvider.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+
+contract DummyLiquidityProvider
+{
+    /// @dev Quotes the amount of `makerToken` that would be obtained by
+    ///      selling `sellAmount` of `takerToken`.
+    /// @param sellAmount Amount of `takerToken` to sell.
+    /// @return makerTokenAmount Amount of `makerToken` that would be obtained.
+    function getSellQuote(
+        address, /* takerToken */
+        address, /* makerToken */
+        uint256 sellAmount
+    )
+        external
+        view
+        returns (uint256 makerTokenAmount)
+    {
+        makerTokenAmount = sellAmount - 1;
```

Comment:
Could be good to use `LibSafeMath` here and below to avoid wonky output in future testing.